### PR TITLE
Adds rstudio server 

### PR DESCRIPTION
**Pull request by tschannik**
### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

**Comment by tschannik** [comment]
I added explanation steps on how to configure the one-click-app to keep persistent data but was not able to get it to work. Any ideas in how to do this successfully?

File: public/v4/apps/rstudio.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+            Enter your RStudio Configuration parameters and click on next. It will take about a minute for the process to finish.
+        end: >-
+            Aaaand you're done! 🔥 
+            Your RStudio Server instance is available at http://$$cap_appname.$$cap_root_domain
+            Use username "rstudio" and your super secret passwort to login to your RStudio Server instance
+
+
+            To keep files after restarts you have to edit your caprover config at /captain/data/config-captain.json
+            and set "hasPersistentData": true
```
Comment (githubsaturn) [inline public/v4/apps/rstudio.yml:35]: You don't need this. All you need to do is to add a persistent directory whose files will be preserved during restarts, something like this:
```
        image: rocker/rstudio:$$cap_rstudio_version
        volumes:
            - $$cap_appname-code:/home/rstudio
```

Also, you may need to add `ROOT=TRUE` as an env var to be able to create and store files in this volume. Maybe, maybe not. Give it a shot either way.

File: public/v4/apps/rstudio.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    displayName: RStudio
+    isOfficial: true
```
Comment (githubsaturn) [inline public/v4/apps/rstudio.yml:37]: I don't think Rocker is official. It's community maintained.

**Review by githubsaturn — COMMENTED** [review]
Overall looks good, just some minor comments.

File: public/v4/apps/rstudio.yml (hunk 2 above)
Comment (tschannik) [reply public/v4/apps/rstudio.yml]: Yep, you're right. Updated the value

File: public/v4/apps/rstudio.yml (hunk 1 above)
Comment (tschannik) [reply public/v4/apps/rstudio.yml]: Works like a charm 👍 thanks

**Comment by githubsaturn** [comment]
Thanks! Looks good! Just a quick validation error to fix:
```
Error: Description too long for rstudio  - keep it below 200 chars
```

**Comment by tschannik** [comment]
@githubsaturn unlucky .. was 204 characters 😅 should be fine now

**Comment by githubsaturn** [comment]
Thanks!

**Pull request merged**